src/pages/public_registration/DeletedPublicationInformation.tsx

```diff
@@ -30,11 +30,11 @@ export const DeletedPublicationInformation = ({ registration }: DeletePublicatio
             sx={{
               display: 'flex',
               justifyContent: 'center',
-              backgroundColor: 'registration.main',
+              backgroundColor: 'info.main',
               padding: '1rem',
               my: '1rem',
             }}>
-            <Typography variant="h2" component="strong">
+            <Typography variant="h2" component="h1" color="white">
```

Comment:
Sikker på at vi vil ha `h1` her? navnet på resultatet er vel allerede i `h1` så det blir sikkert to `h1` i så fall, som ikke er ønskelig 🤔 
Tror `h2` også blir litt rart, og at det er `strong` vi vil ha her, men mulig noe har endret seg fra tidligere